pkg/virt-v2v/customize/scripts/rhel/run/network_config_util.sh

```diff
@@ -299,14 +314,19 @@ check_dupe_hws() {
 
 # Create udev rules check for duplicates and write them to udev file
 main() {
-    {
+   {
         udev_from_ifcfg
         udev_from_nm
         udev_from_netplan
         udev_from_ifquery
     } | check_dupe_hws > "$UDEV_RULES_FILE" 2>/dev/null
-    echo "New udev rule:"
+
+    {
+        udev_final_from_temp_rules
```

Comment:
NP: Could you pipe the data into the function and add it after `check_dupe_hws`?